src/serializer/serializer.js

```diff
@@ -1498,6 +1580,42 @@ export class Serializer {
       }
     }
 
+    // Inject initializer code for indexed vars into functions
+    for (let [functionValue, funcNode] of funcNodes) {
+      let scopeInitializer = this.functionInitializers.get(functionValue);
+      if (scopeInitializer !== undefined) {
+        let scopeInitializerBody = scopeInitializer.body;
+        if (scopeInitializerBody.length === 0) {
+          this.functionInitializers.delete(functionValue);
```

Comment:
It is not obvious to me why you are deleting this. Is functionInitializers used again?